webcompat/static/js/lib/milestones.js

```diff
@@ -0,0 +1,92 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var issues = issues || {}; // eslint-disable-line no-use-before-define
+
+issues.MilestonesView = issues.CategoryView.extend({
+  el: $(".js-Issue-milestones"),
+  keyboardEvents: {
+    m: "openMilestoneEditor"
+  },
+  template: wcTmpl["issue/issue-milestones.jst"],
+  // this subTemplate will need to be kept in sync with
+  // relavant parts in issue/issue-labels.jst
+  subTemplate: wcTmpl["issue/issue-labels-sub.jst"],
+  openMilestoneEditor: function(e) {
+    // make sure we're not typing in the search input.
+    if (e.target.nodeName === "TEXTAREA") {
+      return;
+    } else {
+      e.preventDefault();
+      this.editItems();
+    }
+  },
+  closeEditor: function() {
+    this.milestoneEditor.closeEditor();
+  },
+  fetchItems: function() {
+    var milestonesModel = [
+      { name: "needsdiagnosis", id: 2, state: "open", order: 1 },
+      { name: "wontfix", id: 3, state: "closed", order: 2 }
+    ];
+    this.editorButton = $(".js-CategoryEditorLauncher");
+    this.milestoneEditor = new issues.MilestoneEditorView({
+      model: milestonesModel,
+      issueView: this
+    });
+    if (this._isLoggedIn) {
+      //this.issueLabels = this.getIssueMilestones();
+      this.editorButton.show();
+    }
+  },
+  getIssueMilestones: function() {
+    // TODO: have this do the right thing...
+    return _.pluck(this.model.get("labels"), "name");
```

Comment:
this.model.get("milestones") ?

Reply:
Yeah, I have a follow up issue for that. #1766